privacy.html

```diff
@@ -0,0 +1,88 @@
+user data, except what you voluntarily give to us via email, up on registration or other direct contact from you.
+We do not sell or rent this information to anyone.</p>
+
+<h2>GraphHopper Website and Documentation</h2>
+<p>For web analytics we use <a href="https://matomo.org/">Matomo (piwik)</a> where we do not store the full IP address but only 2 bytes like 192.168.xxx.xxx,
+as well as information like your internet domain, the date and time of your visit, and the pages you 
+access on GraphHopper.com. This information is collected solely for diagnostic and analytic purposes. We do not share any identifiable information about this
+information at any time, unless required by law.
+</p>
+
+<p>
+If you contact us via our contact form, the data is stored in our
+<a href="https://centralstationcrm.com/privacy">CRM system</a> hosted in Germany.
+If you contact us via email, the emails are stored at our email-provider
+<a href="https://www.df.eu/int/datenschutz/">DomainFactory</a>.
+</p>
+
+<h3>GraphHopper Maps</h3>
+GraphHopper Maps is part of the GraphHopper website and so the same policy
+applies. Additionally, we use the default map provider
+<a href="https://omniscale.com/termsofuse">Omniscale</a> where the request is processed
+to provide us with map tiles.
+
+<h3>Cookies</h3>
+<p>We use cookies information to<br>
+(i) simplify browsing our website so you do not need to re-enter user data during your visit or in subsequent visits;<br>
+(ii) maintain sessions for authenticated users; and<br>
+(iii) monitor aggregate metrics such as total number of visitors and pages viewed.</p>
+<p>Cookies are not used for the GraphHopper Directions API.</p>
+
+<h2>GraphHopper Directions API</h2>
+<p>For the GraphHopper Directions API we collect personal information of the customer such as name, email, registration IP,
+time and company name. The purpose is to send technical emails, support
+contact, privacy or term changes, newsletter (only if agreed), the obligation to prove the business nature of the
+registration request and to reduce likelihood of misusage of credit card information at a later stage.
+To avoid newsletter emails, you can unsubscribe via a link provided at the
+end of every newsletter. If you want your account to be removed entirely, send us a message via the dashboard.
+</p>
+<p>
+We process the API requests on rented servers from <a href="https://wiki.hetzner.de/index.php/Datenschutz-FAQ/en">Hetzner</a> hosted in Germany. As the
+geographic coordinates might be personal information we take special care.
+</p>
+
+<p>
+Every API request is stored. We save the associated information (request body and
+headers, IP, time) for a maximum of 5 weeks. We use this to identify
+misusage of API tokens and to
+improve performance and usability of our API.
+Our customers have the choice to use a proxy to enhance the privacy of their
+users.
+</p>
```

Comment:
Should we mention Kurviger here as well? Probably not necessary?

Reply:
Hmmh, as this is an individual, separate contract I don't think we should include it here.